### PR TITLE
Categorizes the scrubber event (and gives it a description)

### PR DESCRIPTION
**Pull request by dragomagol**
## About The Pull Request
image

https://github.com/tgstation/tgstation/pull/68378#issuecomment-1235336597

## Why It's Good For The Game
Consistency mostly

## Changelog

:cl:
fix: scrubber overflow event is now properly categorized and has a description in the trigger event menu
/:cl:

### CODE REVIEW

File: code/modules/events/scrubber_overflow.dm [hunk 1]
```diff
@@ -3,7 +3,9 @@
 	typepath = /datum/round_event/scrubber_overflow
 	weight = 10
 	max_occurrences = 3
-	min_players = 10
+	min_players = 10 MINUTES
+	category = EVENT_CATEGORY_ENGINEERING
```
Comment (JohnFulpWillard) [inline code/modules/events/scrubber_overflow.dm:7]: ``///Events that cause breakages and malfunctions that could be fixed by engineers``
This is more like a Janitor one, maybe this (and scrubber clog) can be moved to a new Janitor one?

**Comment by SuperNovaa41** [comment]
thumbs up means you're doing it i hope so im drafting this

File: code/modules/events/scrubber_overflow.dm [hunk 2]
```diff
@@ -3,7 +3,9 @@
 	typepath = /datum/round_event/scrubber_overflow
 	weight = 10
 	max_occurrences = 3
-	min_players = 10
+	min_players = 10 MINUTES
```
Comment (JohnFulpWillard) [inline code/modules/events/scrubber_overflow.dm:6]: are players supposed to be in minutes?

Comment (dragomagol) [reply code/modules/events/scrubber_overflow.dm:6]: ffff

**Pull request merged**